slices/libpam-modules.yaml

```diff
@@ -0,0 +1,74 @@
+package: libpam-modules
+
+slices:
+  libs:
+    essential:
+      - libpam-modules_config
+      - libc6_libs
+      - libaudit1_libs
+      - libcrypt1_libs
+      - libdb5.3_libs
+      - libpam-modules-bin_base
+      - libpam0g_libs
+      - libselinux1_libs
+    contents:
+      /usr/lib/*-linux-*/security/pam_access.so:
+      /usr/lib/*-linux-*/security/pam_debug.so:
+      /usr/lib/*-linux-*/security/pam_deny.so:
+      /usr/lib/*-linux-*/security/pam_echo.so:
+      /usr/lib/*-linux-*/security/pam_env.so:
+      /usr/lib/*-linux-*/security/pam_exec.so:
+      /usr/lib/*-linux-*/security/pam_extrausers.so:
+      /usr/lib/*-linux-*/security/pam_faildelay.so:
+      /usr/lib/*-linux-*/security/pam_faillock.so:
+      /usr/lib/*-linux-*/security/pam_filter.so:
+      /usr/lib/*-linux-*/security/pam_ftp.so:
+      /usr/lib/*-linux-*/security/pam_group.so:
+      /usr/lib/*-linux-*/security/pam_issue.so:
+      /usr/lib/*-linux-*/security/pam_keyinit.so:
+      /usr/lib/*-linux-*/security/pam_lastlog.so:
+      /usr/lib/*-linux-*/security/pam_limits.so:
+      /usr/lib/*-linux-*/security/pam_listfile.so:
+      /usr/lib/*-linux-*/security/pam_localuser.so:
+      /usr/lib/*-linux-*/security/pam_loginuid.so:
+      /usr/lib/*-linux-*/security/pam_mail.so:
+      /usr/lib/*-linux-*/security/pam_mkhomedir.so:
+      /usr/lib/*-linux-*/security/pam_motd.so:
+      /usr/lib/*-linux-*/security/pam_namespace.so:
+      /usr/lib/*-linux-*/security/pam_nologin.so:
+      /usr/lib/*-linux-*/security/pam_permit.so:
+      /usr/lib/*-linux-*/security/pam_pwhistory.so:
+      /usr/lib/*-linux-*/security/pam_rhosts.so:
+      /usr/lib/*-linux-*/security/pam_rootok.so:
+      /usr/lib/*-linux-*/security/pam_securetty.so:
+      /usr/lib/*-linux-*/security/pam_selinux.so:
+      /usr/lib/*-linux-*/security/pam_sepermit.so:
+      /usr/lib/*-linux-*/security/pam_setquota.so:
+      /usr/lib/*-linux-*/security/pam_shells.so:
+      /usr/lib/*-linux-*/security/pam_stress.so:
+      /usr/lib/*-linux-*/security/pam_succeed_if.so:
+      /usr/lib/*-linux-*/security/pam_time.so:
+      /usr/lib/*-linux-*/security/pam_timestamp.so:
+      /usr/lib/*-linux-*/security/pam_tty_audit.so:
+      /usr/lib/*-linux-*/security/pam_umask.so:
+      /usr/lib/*-linux-*/security/pam_unix.so:
+      /usr/lib/*-linux-*/security/pam_userdb.so:
+      /usr/lib/*-linux-*/security/pam_usertype.so:
+      /usr/lib/*-linux-*/security/pam_warn.so:
+      /usr/lib/*-linux-*/security/pam_wheel.so:
+      /usr/lib/*-linux-*/security/pam_xauth.so:
+
+  config:
+    contents:
+      /etc/security/access.conf:
+      /etc/security/faillock.conf:
+      /etc/security/group.conf:
+      /etc/security/limits.conf:
+      /etc/security/namespace.conf:
+      /etc/security/namespace.init:
+      /etc/security/pam_env.conf:
+      /etc/security/sepermit.conf:
+      /etc/security/time.conf:
+      /etc/security/limits.d/:
+      /etc/security/namespace.d/:
+      /usr/share/pam-configs/mkhomedir:
```

Comment:
Files under `/usr/share/pam-configs` are shipped by various packages - It's loaded by pam-auth-update which actually is not used at all since pam-auth-update relies on debconf environment, which is not available. Instead in core we ship static pam files so we can control the contents.

I'll move these files to their own slice and not reference to them